doc/source/api/core/solver/fielddata.rst

```diff
@@ -2,103 +2,81 @@
 
 Field Data
 ==========
-The ``FieldData`` module provides access to Fluent surface, scalar and vector field
-data. It provides two sets of APIs.
+The ``FieldData`` module provides access to Fluent surface, scalar, and vector field
+data. It has two APIs.
 
-#. APIs to get multiple fields in a single request. 
-#. APIs to get one field per request. 
+- An API to get multiple fields in a single request 
+- An API to get one field per request 
```

Comment:
In fact, use of the term _API(s)_ unnecessarily shrouds this in mystery. It's only when you scroll down the web page that you realise that it's talking about (Python class) methods and that the field_data that you access is actually an attribute of a session, and therefore an object. The page only talks about field data as 'module' that has 'APIs'.  If you deleted all the existing guff and just left the example, it would actually be an improvement.